backup-stores/azure/src/main/java/io/camunda/zeebe/backup/azure/AzureBackupStore.java

```diff
@@ -24,16 +31,58 @@
  * scheme: {@code basePath/partitionId/checkpointId/nodeId}.
  */
 public final class AzureBackupStore implements BackupStore {
-
-  private final AzureBackupConfig azureBackupConfig;
+  public static final String SNAPSHOT_FILESET_NAME = "snapshot";
+  public static final String SEGMENTS_FILESET_NAME = "segments";
+  private final ExecutorService executor;
+  private final FileSetManager fileSetManager;
+  private final ManifestManager manifestManager;
```

Comment:
:x: The executor needs to be closed
:x: The azure client is being passed to the fileset and manifest managers and not being closed.